File: Day_1.py
import math

def mass_to_fuel_req(masses):

    file = open(masses, "r")
    data = file.readlines()
    list_to_combine = []
    for line in data:
        mass = line.split()
        divided = int(mass[0]) / 3
        #Not Satisfied with how I extracted information from the file
        floored = math.floor(divided)
        deduct = floored - 2
        list_to_combine.append(deduct)

    total = sum(list_to_combine)
    return total

    file.close()

print(mass_to_fuel_req("Puzzle_Input_1.txt"))

File: day1.cpp
#include <iostream>
#include <fstream>
#include <string>
#include <cmath>
using namespace std;

//Part 1
int main()
{
    ifstream inFile;
    inFile.open("Puzzle_Input_Day_1.txt");
    int num;
    int sum = 0;
    while (inFile >> num)
    {
        int fuel = floor(num/3) - 2;
        sum += fuel;
    }
    cout << sum << endl;
    inFile.close();
    return 0;
    
}
